aleph/views/xref_api.py

```diff
@@ -53,6 +61,13 @@ def index(collection_id):
     """
     get_index_collection(collection_id, request.authz.READ)
     result = XrefQuery.handle(request, collection_id=collection_id)
+    if get_flag("party", default=False):
+        # In party mode we merge in some random xref pairs and shuffle the
+        # collection. This gives us a good mix of positive and negative matches
+        results_party = XrefPartyQuery.handle(request, collection_id=collection_id)
+        results_merged = result.results + results_party.result
```

Comment:
This means we're returning twice as many results as requested in the query. I'm a bit concerned this is going to confuse the frontend, but we can try.

Reply:
@kjacks thoughts?